### DISCUSSION TITLE
Cannot search date in my locale

### DISCUSSION THREAD

**Discussion by Pamposgsk** [laravel/nova-issues#3697] [Q&A]
Hi,

<img width="1354" alt="Screenshot 2022-01-03 at 11 42 12 PM" src="https://user-images.githubusercontent.com/8403769/147983599-9a7d846d-6559-4686-8f48-a656a61ad7f7.png">

Let say that I am trying to find the first row by searching the birthdate .
If my search is: 2021-09-08 I will successfully find the patient which is the first row on the above image
If I search the date in my locale which is 08-09-2021 it will not find any result.

How can I overcome this issue ?

Thanks 
Pampos

**Comment by danielschweiger** [comment]
I think it works because "YYYY-MM-DD" is also the format in which the dates are saved in your database. 

For date filtering you should use a filter, for example:
https://github.com/pos-lifestyle/laravel-nova-date-range-filter

This package works great for me.
